### DISCUSSION TITLE
Help with RuntimeError

### DISCUSSION THREAD

**Discussion by katenautiyal** [CoBrALab/RABIES#364] [General]
Hi, I can successfully run the test data, but run into this error when running other data. Can't figure it out, do you have any ideas? 

 File "/home/rabies/miniforge/lib/python3.9/site-packages/SimpleITK/SimpleITK.py", line 5779, in Execute
    return _SimpleITK.ImageFileReader_Execute(self)
RuntimeError: Exception thrown in SimpleITK ImageFileReader_Execute: /home/rabies/miniforge/include/ITK-5.1/itkImageBase.hxx:184:
itk::ERROR: itk::ERROR: Image(0x557ca20eba50): Bad direction, determinant is 0. Direction is -0 0 0
0 -0 0
0 0 0

**Comment by gdevenyi** [comment]
This is insufficent context to answer your question. Please provide all the information requested in the issue template.

**Comment by katenautiyal** [comment]
Thanks, sorry - here is more detail!

**Describe the bug**
I don't think it's actually a bug, I'm just stuck trying to process some data.

**Describe RABIES call**
I'm running RABIES latest in Docker Desktop on Windows. I'm using a command shell to run:

docker run -it --rm -v .\token_foo:/token_foo:ro -v .\token_rabies_outputs:/home/rabies/token_rabies_outputs gabdesgreg/rabies:latest -p MultiProc --local_threads 6 preprocess /token_foo/ ./token_rabies_outputs --anat_autobox --bold_autobox --apply_despiking --HMC_option 2 --anat_inho_cor method=Affine,otsu_thresh=2,multiotsu=true --anat_robust_inho_cor apply=true,masking=true,brain_extraction=true,template_registration=Affine --bold_robust_inho_cor apply=true,masking=true,brain_extraction=true,template_registration=Affine --bold2anat_coreg masking=true,brain_extraction=true,registration=Affine

I've tried this command many different ways, including using all defaults, and it always results in the same error. I can process the test data from the tutorial just fine (https://zenodo.org/records/8349029) but when I try to run this Mouse_rest_psilocybin data (https://openneuro.org/datasets/ds001725/versions/1.1.0/download) I get the image read error when doing preprocessing. Before processing the data I am deleting the duplicate sub data that's in the Mouse_rest_psilocybin download. Here is a single subject example (with the dupes removed) from Mouse_rest_psilocybin, that runs into this problem. ((https://www.dropbox.com/scl/fi/eoacb5fwqbe46a7v64yrv/sub-001.zip?rlkey=ow7xnjnq3cou6ou6g3vmrmsmq&dl=0)).

**Attach log file**
Attached

**Attach QC_report**
There is no QC_report, I think because preprocessing failed (is that right?)

**Additional context**
Thanks for any help, I really appreciate it!
[rabies_preprocess.log](https://github.com/CoBrALab/RABIES/files/14566243/rabies_preprocess.log)

**Comment by Gab-D-G** [comment]
Something is wrong with your anatomical image, I cannot open it with the ANTs/ITK tools:
```
~/Downloads/sub-001/ses-1/anat$ PrintHeader sub-001_ses-1_acq-RARE_T2w.nii.gz
 cant read sub-001_ses-1_acq-RARE_T2w.nii.gz
```
Can you inspect this file's metadata? I don't think I can tell what is going on if I cannot inspect it.
